src/app/static/src/app/store/load/mutations.ts

```diff
@@ -66,10 +67,13 @@ export const mutations: MutationTree<LoadState> & LoadMutations = {
     },
     PreparingRehydrate(state: LoadState, action: PayloadWithType<ProjectRehydrateSubmitResponse>) {
         state.rehydrateId = action.payload.id;
-        state.preparing = true;
         state.complete = false;
         state.loadError = null;
     },
+    StartPreparingRehydrate(state: LoadState, action: PayloadWithType<boolean>) {
```

Comment:
Does this actually need a payload? It never gets called with `false` does it, so can we not just assume that it we're starting preparation we set `preparing` to `true`?